plugins/auth-backend/src/lib/oauth/OAuthAdapter.ts

```diff
@@ -110,9 +102,11 @@ export class OAuthAdapter implements AuthProviderRouteHandlers {
       throw new InputError('No env provided in request query parameters');
     }
 
+    const cookieConfig = this.getCookieConfig();
```

Comment:
We have the origin here right? It's grabbed from `req.query` above. It's very important that the nonce cookie is set correctly too, or auth will fail

Reply:
Ah yes, good catch! I wasn't sure if that's what we wanted to do 👍